### PR TITLE
MINOR: Add missing apiversion test for 3.0

### PR DESCRIPTION
**Pull request by dengziming**
*More detailed description of your change*
#10504 bump apiversion to 3.0.0, we'd better also add test for it.

*Summary of testing strategy (including rationale)*
Unit test

### CODE REVIEW

**Comment by dengziming** [comment]
ping @mumrah for a review, maybe you forgot it.

**Review by showuon — APPROVED** [review]
LGTM!

**Comment by dengziming** [comment]
I also find #10760 bumped the IBP to V3.0.1 but forgot to add a test for it. ping @thomaskwscott  @dajac to have a look.

**Comment by thomaskwscott** [comment]
> I also find #10760 bumped the IBP to V3.0.1 but forgot to add a test for it. ping @thomaskwscott @dajac to have a look.

thanks for the catch @dengziming, lgtm

**Comment by dajac** [comment]
@dengziming Good catch!

File: core/src/test/scala/unit/kafka/api/ApiVersionTest.scala [hunk 1]
```diff
@@ -167,6 +171,8 @@ class ApiVersionTest {
     assertEquals("2.6", KAFKA_2_6_IV0.shortVersion)
     assertEquals("2.7", KAFKA_2_7_IV2.shortVersion)
     assertEquals("2.8", KAFKA_2_8_IV0.shortVersion)
+    assertEquals("3.0", KAFKA_3_0_IV0.shortVersion)
```
Comment (dajac) [inline core/src/test/scala/unit/kafka/api/ApiVersionTest.scala:175]: Should 2.8 IV1 be tested here as well?

**Review by dajac — COMMENTED** [review]
Left a small comment.

File: core/src/test/scala/unit/kafka/api/ApiVersionTest.scala (hunk 1 above)
Comment (dengziming) [reply core/src/test/scala/unit/kafka/api/ApiVersionTest.scala:175]: Done

**Review by dajac — APPROVED** [review]
LGTM

**Pull request merged**